models/createModel.js

```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
+const validator = require('validator');
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+    name: {
+        type: String,
+        required: [true, 'Name is Required'],
+    },
+    password: {
+        type: String,
+        required: [true, 'Password is Required'],
+        minlength: 6
+    },
+    email: {
+        type: String,
+        unique: true,
+        required: [true, 'Email is Required'],
+        validate: {
+            validator: email => validator.isEmail(email),
+            message: '{VALUE} is not a valid email'
+        }
+    },
+    mobile: {
+        type: String,
+        validate: {
+            validator: function (v) {
+                var re = /^\d{10}$/;
+                return (v == null || v.trim().length < 1) || re.test(v)
+            },
+            message: '{VALUE} is not a valid mobile number'
+        }
+    },
+    address: {
+        type: String,
+        required: [true, 'Address is Required']
+    }
```

Comment:
Add a createdOn field default to current timestamp.

Reply:
Added.